lib/omniauth/onelogin_setup.rb

```diff
@@ -0,0 +1,41 @@
+module OneloginSetup
+  def self.configure(builder)
+    public_key = ENV.fetch('GOVUK_LOGIN_PUBLIC_KEY', 'something')
+    onelogin_issuer_uri = URI(ENV.fetch('GOVUK_LOGIN_ISSUER_URL', 'https://website.gov.uk/'))
+    private_key_pem = ENV.fetch('GOVUK_LOGIN_PRIVATE_KEY', 'something')
+
+    private_key_pem = private_key_pem.gsub('\n', "\n")
+    host_env = HostingEnvironment.application_url
+
+    begin
+      private_key = OpenSSL::PKey::RSA.new(private_key_pem)
+      Rails.logger.debug 'RSA private key successfully created.'
+    rescue OpenSSL::PKey::RSAError => e
+      Rails.logger.debug { "Failed to create RSA private key: #{e.message}" }
+    end
```

Comment:
Is this code intended to allow developers to run the code without properly setting up the ENV vars?